checkov/terraform/checks/resource/oci/SecurityGroupsIngressStatelessSecurityRules.py

```diff
@@ -0,0 +1,23 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class SecurityGroupsIngressStatelessSecurityRules(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure security groups has stateless ingress security rules"
```

Comment:
```suggestion
        name = "Ensure security group has stateless ingress security rules"
```
or 
```suggestion
        name = "Ensure security groups have stateless ingress security rules"
```
your choice 🙂